python/pyspark/sql/types.py

```diff
@@ -1624,6 +1624,50 @@ def to_arrow_type(dt):
     return arrow_type
 
 
+def to_arrow_schema(schema):
+    """ Convert a schema from Spark to Arrow
+    """
+    import pyarrow as pa
+    fields = [pa.field(field.name, to_arrow_type(field.dataType)) for field in schema]
```

Comment:
We should add `nullable=field.nullable` just in case?

Reply:
yeah, good idea